biscuit-auth/src/token/authorizer.rs

```diff
@@ -914,29 +947,33 @@ impl<'t> Authorizer<'t> {
 
     /// returns all of the data loaded in the authorizer
     pub fn dump(&self) -> (Vec<Fact>, Vec<Rule>, Vec<Check>, Vec<Policy>) {
-        let mut checks = self.checks.clone();
+        let mut checks = self.authorizer_block_builder.checks.clone();
         checks.extend(
             self.token_checks
                 .iter()
                 .flatten()
                 .map(|c| Check::convert_from(c, &self.symbols).unwrap()),
         );
 
-        (
-            self.world
-                .facts
-                .iter_all()
-                .map(|f| Fact::convert_from(f.1, &self.symbols))
-                .collect::<Result<Vec<_>, error::Format>>()
-                .unwrap(),
-            self.world
-                .rules
-                .iter_all()
-                .map(|r| Rule::convert_from(r.1, &self.symbols).unwrap())
-                .collect(),
-            checks,
-            self.policies.clone(),
-        )
+        let mut facts = self
+            .world
+            .facts
+            .iter_all()
+            .map(|f| Fact::convert_from(f.1, &self.symbols))
+            .collect::<Result<Vec<_>, error::Format>>()
+            .unwrap();
```

Comment:
We might want to return a Vec<Result> knqtead: if there was an invalid fact for any reason, we would like to see the valid facts and the errors

Reply:
yeah. for authorization, we want an error to blow up the whole computation, but for querying i guess i can see the merit in ignoring errors.

Reply:
Let's decide this in the "revamp the authorizer lifecycle", since it will touch queries as well.